stdlib/map.ml

```diff
@@ -56,6 +56,7 @@ module type S =
     val partition: (key -> 'a -> bool) -> 'a t -> 'a t * 'a t
     val split: key -> 'a t -> 'a t * 'a option * 'a t
     val is_empty: 'a t -> bool
+    val is_singleton: 'a t -> bool
```

Comment:
This part has been merged in #14118, so this part of the patch is no longer relevant.